.travis/deploy_relational_database_shredder.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+tag=$1
+
+project_name="relational_database_shredder/"
+project_name_len=${#project_name}
```

Comment:
@chuwy curious as to what you think about that, that would remove the length of the beginning of the tag being hardcoded?